app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,98 @@
+class ReviewsController < ApplicationController
+  before_action :set_review, only: [:show, :edit, :update, :destroy]
+  # GET /reviews
+  # GET /reviews.json
+  def index
+    @reviews = Review.all
+  end
+
+  # GET /reviews/1
+  # GET /reviews/1.json
+  def show
+  end
+
+  # GET /reviews/new
+  def new
+    @restaurant_id = params[:restaurant_id]
+    @review = Review.new
+
+  end
+
+  # GET /reviews/1/edit
+  def edit
+  end
+
+  # POST /reviews
+  # POST /reviews.json
+  def create
+    @review = Review.new(review_params)
+    restaurant_id = review_params[:restaurant_id]
```

Comment:
do you really need this in a variable?
you could use `@review.restaurant_id` instead